### ISSUE TITLE
Status bar icons are not visible in white theme on Android 5.0 device

### ISSUE THREAD

**Issue by kafumi** [DroidKaigi/conference-app-2019#392]
**Overview (Required)**
- Status bar icons are not visible in white theme screens on Android 5.0 device
- It's because they are white and the background of status bar area is also white in white theme

**Steps To Reproduce**
1. Launch app on Android 5.0 device
2. Go to "About" screen

**Expected behavior**
Status bar icons are visible in all screens.

**Actual behavior**
Status bar icons are not visible in white theme screens.

**Screenshots**
<img width="300" src="https://user-images.githubusercontent.com/12084705/51069693-427f0880-165a-11e9-85ff-725189831114.png" />

**Env:**
 - Device: Android emulator x86_64
 - OS: Android 5.0
 - App Version or the branch: master 26297f258f38461adbfba84bf1127ec34ba3e691

**Additional context**
This issue is probably caused because changing the color of status bar icons is not capable in API level 22 or lower

```kotlin
// MainActivity.kt
        navController.addOnDestinationChangedListener { _, destination, _ ->
            val config = PageConfiguration.getConfiguration(destination.id)
            // ...
            if (23 <= Build.VERSION.SDK_INT) {
                window.decorView.systemUiVisibility = if (config.isWhiteTheme) {
                    View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
                } else {
                    0
                }
            }
```

**Comment by kafumi** [comment]
Let me propose a possible fix along with #269.

**Comment by takahirom** [comment]
I don't notice it! Good catch! 🐱
